bundles/org.openhab.binding.speedtest/src/main/java/org/openhab/binding/speedtest/internal/SpeedtestHandler.java

```diff
@@ -82,6 +97,7 @@ public class SpeedtestHandler extends BaseThingHandler {
     private String interfaceExternalIp = "";
     private String resultUrl = "";
     private String server = "";
+    private String timestamp = "";
```

Comment:
Please think about typing correctly already here.

Reply:
reworked together with the above change